app/src/test/java/com/courseproject/tindar/controllers/viewprofiles/ViewProfilesControllerUnitTest.java

```diff
@@ -0,0 +1,38 @@
+package com.courseproject.tindar.controllers.viewprofiles;
+
+import static org.junit.Assert.assertEquals;
+
+import com.courseproject.tindar.usecases.viewprofiles.ViewProfilesDsResponseModel;
+import com.courseproject.tindar.usecases.viewprofiles.ViewProfilesInputBoundary;
+
+import org.junit.Test;
+
+import java.util.Date;
+import java.util.GregorianCalendar;
+
+public class ViewProfilesControllerUnitTest {
```

Comment:
Again, you need to input mockUserListUserInput as an argument when instantiating controller, and call the method in the controller to unit test controller